src/window_identifier/wayland.rs

```diff
@@ -24,25 +24,9 @@ const ZXDG_EXPORTER_V2: u32 = 1;
 
 #[derive(Debug)]
 pub struct WaylandWindowIdentifier {
-    exported: Exported,
     type_: WindowIdentifierType,
 }
 
-#[derive(Debug)]
-enum Exported {
-    V1(ZxdgExportedV1),
-    V2(ZxdgExportedV2),
-}
-
-impl Exported {
```

Comment:
I feel like we should provide some way for the user to destroy those whenever they want